docs/versioned_docs/version-v1.0.0-beta.2/how_to/how-to-oracles.md

```diff
@@ -267,6 +267,19 @@ app.use(cors())
 
 :::
 
+## Foreign Calls
+
+When making foreign calls, you need to ensure that:
+
+1. The function is marked with the `unconstrained` keyword
+2. The function is properly defined in your backend implementation
+
```

Comment:
this was only half deleted

Reply:
thanks, sorry for the back and forth, did it in a hurry and missed most of it